[2025-06-14T23:49:27.779-0400] {processor.py:157} INFO - Started process (PID=40393) to work on /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:49:27.782-0400] {processor.py:829} INFO - Processing file /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-14T23:49:27.782-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:49:27.782-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:49:27.826-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:49:28.175-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:49:28.175-0400] {manager.py:499} INFO - Created Permission View: %s
[2025-06-14T23:49:28.180-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:49:28.180-0400] {manager.py:499} INFO - Created Permission View: %s
[2025-06-14T23:49:28.182-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:49:28.182-0400] {manager.py:499} INFO - Created Permission View: %s
[2025-06-14T23:49:28.182-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:49:28.182-0400] {dag.py:2941} INFO - Sync 1 DAGs
[2025-06-14T23:49:28.186-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:49:28.186-0400] {dag.py:2963} INFO - Creating ORM DAG for example_local_kubernetes_executor
[2025-06-14T23:49:28.186-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:49:28.186-0400] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2025-06-14T23:49:28.193-0400] {processor.py:179} INFO - Processing /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.414 seconds
[2025-06-14T23:55:41.260-0400] {processor.py:157} INFO - Started process (PID=46004) to work on /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:55:41.263-0400] {processor.py:829} INFO - Processing file /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-14T23:55:41.263-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:55:41.263-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:55:41.281-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:55:41.281-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-06-14T23:55:41.281-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:55:41.281-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-06-14T23:55:41.281-0400] {processor.py:841} WARNING - No viable dags retrieved from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:55:41.325-0400] {processor.py:179} INFO - Processing /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2025-06-14T23:56:43.219-0400] {processor.py:157} INFO - Started process (PID=47514) to work on /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:56:43.222-0400] {processor.py:829} INFO - Processing file /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-14T23:56:43.222-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:56:43.222-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:56:43.240-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:56:43.240-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-06-14T23:56:43.240-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:56:43.240-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-06-14T23:56:43.240-0400] {processor.py:841} WARNING - No viable dags retrieved from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:56:43.285-0400] {processor.py:179} INFO - Processing /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2025-06-14T23:57:22.973-0400] {processor.py:157} INFO - Started process (PID=48500) to work on /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:57:22.976-0400] {processor.py:829} INFO - Processing file /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-14T23:57:22.976-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:57:22.976-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:57:22.996-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:57:22.996-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-06-14T23:57:22.996-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:57:22.996-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-06-14T23:57:22.996-0400] {processor.py:841} WARNING - No viable dags retrieved from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:57:23.045-0400] {processor.py:179} INFO - Processing /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.073 seconds
[2025-06-14T23:57:36.442-0400] {processor.py:157} INFO - Started process (PID=48656) to work on /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:57:36.444-0400] {processor.py:829} INFO - Processing file /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-14T23:57:36.445-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:57:36.445-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:57:36.463-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:57:36.463-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-06-14T23:57:36.463-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:57:36.463-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-06-14T23:57:36.463-0400] {processor.py:841} WARNING - No viable dags retrieved from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:57:36.508-0400] {processor.py:179} INFO - Processing /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2025-06-14T23:58:23.980-0400] {processor.py:157} INFO - Started process (PID=49360) to work on /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:58:23.983-0400] {processor.py:829} INFO - Processing file /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-14T23:58:23.984-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:58:23.984-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:58:24.005-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:58:24.005-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-06-14T23:58:24.006-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:58:24.005-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-06-14T23:58:24.006-0400] {processor.py:841} WARNING - No viable dags retrieved from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:58:24.057-0400] {processor.py:179} INFO - Processing /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.078 seconds
[2025-06-14T23:58:58.556-0400] {processor.py:157} INFO - Started process (PID=50231) to work on /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:58:58.559-0400] {processor.py:829} INFO - Processing file /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-14T23:58:58.559-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:58:58.559-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:58:58.579-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:58:58.579-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-06-14T23:58:58.579-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:58:58.579-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-06-14T23:58:58.579-0400] {processor.py:841} WARNING - No viable dags retrieved from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:58:58.630-0400] {processor.py:179} INFO - Processing /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.075 seconds
[2025-06-14T23:59:12.647-0400] {processor.py:157} INFO - Started process (PID=50410) to work on /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:59:12.649-0400] {processor.py:829} INFO - Processing file /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-14T23:59:12.649-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:59:12.649-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:59:12.668-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:59:12.668-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-06-14T23:59:12.669-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:59:12.669-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-06-14T23:59:12.669-0400] {processor.py:841} WARNING - No viable dags retrieved from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:59:12.718-0400] {processor.py:179} INFO - Processing /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.072 seconds
[2025-06-14T23:59:45.634-0400] {processor.py:157} INFO - Started process (PID=51004) to work on /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:59:45.637-0400] {processor.py:829} INFO - Processing file /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-14T23:59:45.637-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:59:45.637-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:59:45.657-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:59:45.657-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-06-14T23:59:45.657-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:59:45.657-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-06-14T23:59:45.657-0400] {processor.py:841} WARNING - No viable dags retrieved from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:59:45.708-0400] {processor.py:179} INFO - Processing /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.074 seconds
[2025-06-14T23:59:57.440-0400] {processor.py:157} INFO - Started process (PID=51176) to work on /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:59:57.442-0400] {processor.py:829} INFO - Processing file /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-14T23:59:57.442-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:59:57.442-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:59:57.463-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:59:57.462-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-06-14T23:59:57.463-0400] {logging_mixin.py:154} INFO - [2025-06-14T23:59:57.463-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-06-14T23:59:57.463-0400] {processor.py:841} WARNING - No viable dags retrieved from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-14T23:59:57.511-0400] {processor.py:179} INFO - Processing /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.072 seconds
[2025-06-15T00:00:35.461-0400] {processor.py:157} INFO - Started process (PID=51699) to work on /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:00:35.464-0400] {processor.py:829} INFO - Processing file /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-15T00:00:35.464-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:00:35.464-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:00:35.484-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:00:35.484-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-06-15T00:00:35.484-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:00:35.484-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-06-15T00:00:35.484-0400] {processor.py:841} WARNING - No viable dags retrieved from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:00:35.534-0400] {processor.py:179} INFO - Processing /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.073 seconds
[2025-06-15T00:00:43.560-0400] {processor.py:157} INFO - Started process (PID=51843) to work on /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:00:43.563-0400] {processor.py:829} INFO - Processing file /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-15T00:00:43.563-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:00:43.563-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:00:43.583-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:00:43.583-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-06-15T00:00:43.584-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:00:43.584-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-06-15T00:00:43.584-0400] {processor.py:841} WARNING - No viable dags retrieved from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:00:43.637-0400] {processor.py:179} INFO - Processing /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.077 seconds
[2025-06-15T00:01:22.363-0400] {processor.py:157} INFO - Started process (PID=52482) to work on /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:01:22.366-0400] {processor.py:829} INFO - Processing file /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-15T00:01:22.367-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:01:22.367-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:01:22.386-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:01:22.386-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-06-15T00:01:22.386-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:01:22.386-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-06-15T00:01:22.386-0400] {processor.py:841} WARNING - No viable dags retrieved from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:01:22.438-0400] {processor.py:179} INFO - Processing /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.076 seconds
[2025-06-15T00:01:24.545-0400] {processor.py:157} INFO - Started process (PID=52493) to work on /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:01:24.548-0400] {processor.py:829} INFO - Processing file /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-15T00:01:24.548-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:01:24.548-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:01:24.567-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:01:24.567-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-06-15T00:01:24.567-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:01:24.567-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-06-15T00:01:24.567-0400] {processor.py:841} WARNING - No viable dags retrieved from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:01:24.616-0400] {processor.py:179} INFO - Processing /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.072 seconds
[2025-06-15T00:02:07.704-0400] {processor.py:157} INFO - Started process (PID=53153) to work on /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:02:07.706-0400] {processor.py:829} INFO - Processing file /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-15T00:02:07.706-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:02:07.706-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:02:07.727-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:02:07.726-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-06-15T00:02:07.727-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:02:07.727-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-06-15T00:02:07.727-0400] {processor.py:841} WARNING - No viable dags retrieved from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:02:07.777-0400] {processor.py:179} INFO - Processing /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.074 seconds
[2025-06-15T00:02:16.648-0400] {processor.py:157} INFO - Started process (PID=53312) to work on /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:02:16.650-0400] {processor.py:829} INFO - Processing file /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-15T00:02:16.650-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:02:16.650-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:02:16.670-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:02:16.669-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-06-15T00:02:16.670-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:02:16.670-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-06-15T00:02:16.670-0400] {processor.py:841} WARNING - No viable dags retrieved from /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:02:16.720-0400] {processor.py:179} INFO - Processing /Users/rah/Desktop/creative_projects/rah_coding_repo/projects/fin_data_project_001/finance-data-project-1/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.073 seconds
[2025-06-15T00:07:07.745-0400] {processor.py:157} INFO - Started process (PID=57357) to work on /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:07:07.748-0400] {processor.py:829} INFO - Processing file /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-15T00:07:07.748-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:07:07.748-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:07:07.791-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:07:08.111-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:07:08.111-0400] {dag.py:2941} INFO - Sync 1 DAGs
[2025-06-15T00:07:08.115-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:07:08.115-0400] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2025-06-15T00:07:08.123-0400] {processor.py:179} INFO - Processing /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.378 seconds
[2025-06-15T00:08:34.958-0400] {processor.py:157} INFO - Started process (PID=58633) to work on /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:08:34.961-0400] {processor.py:829} INFO - Processing file /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-15T00:08:34.961-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:08:34.961-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:08:35.002-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:08:35.093-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:08:35.093-0400] {dag.py:2941} INFO - Sync 1 DAGs
[2025-06-15T00:08:35.097-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:08:35.097-0400] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2025-06-15T00:08:35.105-0400] {processor.py:179} INFO - Processing /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.148 seconds
[2025-06-15T00:10:18.089-0400] {processor.py:157} INFO - Started process (PID=60534) to work on /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:10:18.092-0400] {processor.py:829} INFO - Processing file /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-15T00:10:18.092-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:10:18.092-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:10:18.136-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:10:18.229-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:10:18.229-0400] {dag.py:2941} INFO - Sync 1 DAGs
[2025-06-15T00:10:18.233-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:10:18.233-0400] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2025-06-15T00:10:18.241-0400] {processor.py:179} INFO - Processing /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.152 seconds
[2025-06-15T00:11:57.740-0400] {processor.py:157} INFO - Started process (PID=61715) to work on /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:11:57.743-0400] {processor.py:829} INFO - Processing file /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-15T00:11:57.743-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:11:57.743-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:11:57.785-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:11:57.881-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:11:57.881-0400] {dag.py:2941} INFO - Sync 1 DAGs
[2025-06-15T00:11:57.885-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:11:57.885-0400] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2025-06-15T00:11:57.893-0400] {processor.py:179} INFO - Processing /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.153 seconds
[2025-06-15T00:13:41.809-0400] {processor.py:157} INFO - Started process (PID=63564) to work on /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:13:41.812-0400] {processor.py:829} INFO - Processing file /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-15T00:13:41.812-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:13:41.812-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:13:41.856-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:13:41.950-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:13:41.950-0400] {dag.py:2941} INFO - Sync 1 DAGs
[2025-06-15T00:13:41.954-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:13:41.954-0400] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2025-06-15T00:13:41.962-0400] {processor.py:179} INFO - Processing /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.154 seconds
[2025-06-15T00:15:22.360-0400] {processor.py:157} INFO - Started process (PID=65423) to work on /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:15:22.363-0400] {processor.py:829} INFO - Processing file /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-06-15T00:15:22.363-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:15:22.363-0400] {dagbag.py:536} INFO - Filling up the DagBag from /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:15:22.407-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-06-15T00:15:22.499-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:15:22.499-0400] {dag.py:2941} INFO - Sync 1 DAGs
[2025-06-15T00:15:22.503-0400] {logging_mixin.py:154} INFO - [2025-06-15T00:15:22.503-0400] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2025-06-15T00:15:22.511-0400] {processor.py:179} INFO - Processing /Users/rah/.pyenv/versions/3.10.11/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.151 seconds
